templates/pages/page.html.twig

```diff
@@ -398,14 +398,11 @@
 
                                             </div>
 
-                                            {#
-                                            <div class="govuk-grid-column-one-third apollo-enclosure">
-                                                <h3 class="govuk-heading-s">Talk to us</h3>
-                                                <p>CCS customer services team is available Monday to Friday, 9am to 5pm.</p>
-
-                                                <h3 class="govuk-heading-s" style="margin-bottom: 0;">Talk to us</h3>
-                                                <p>0345 410 2222</p>
-                                            </div>#}
+                                             <div class="govuk-grid-column-one-third" ;  style=" padding:18px; background:#f8f8f8;:font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 23px" >
+								                <P class="govuk-!-font-size-14px"><b>Talk to us </b></P><br />
+								                <P >CCS customer services team is available <br />Monday to Friday, 9am to 5pm.</P><br />
```

Comment:
There shouldn't be unnecessary spaces in HTML elements.

Instead of:

`<P >`

It should be:

`<p>`

(lowercase the HTML tags and remove unnecessary whitespace)